docs/source/examples/workspace/linchpin.conf

```diff
@@ -111,6 +111,11 @@ distill_on_error = False
 # How long to wait before failing (in seconds) for an async task.
 #async_timeout = 1000
 
+# enables debug mode, which turns on debugging outputs for certain commands.
+# Note that this feature may cause secret keys to be leaked and therefore
+# should not be used in production.
+# _debug_mode = False
```

Comment:
For consistency, you'll notice the actual values (even though they are commented) do not have a space, while the descriptions do. Adjusting this will help.